src/spot_skills/src/spot_skills/spot_manager.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, client_name: str, hostname: str, username: str, password: str
         self.time_sync = SpotTimeSync(self._robot)
 
         self.log_info("Time sync has been established with Spot.")
-        self.log_sync_info()
+        self.resync_and_log_info()
```

Comment:
This function should allow the `TimeSync` member variable to become private once more.

Reply:
Not true. The arm controller needs to access a few different values (e.g., round trip time) to compute when exactly to send out the trajectory segments.